### ISSUE TITLE
Test-AdfLinkedService Class - Object issue cannot be found on this object

### ISSUE THREAD

**Issue by aam-a** [Azure-Player/azure.datafactory.tools#382]
Currently trying to build a Powershell script to test connection of ADF Linked Services -using the Test-AdfLinkedService class for one of the two examples:

$params = @{
DataFactoryName = 'xxxxxx'
ResourceGroupName = 'xxxxxxx'
SubscriptionID = 'xxxxxxx'

}

Example 1
$LinkedServiceName = 'LinkedServiceName'
$r = Test-AdfLinkedService @params -LinkedServiceName $LinkedServiceName
$r | Format-Table

Example 2
$LinkedServiceName = 'LinkedServiceName' # Comma-separated list
$r = Test-AdfLinkedService @params -LinkedServiceName $LinkedServiceName
$r.Report

Both failure due to The property 'succeeded' cannot be found on this object. Verify that the property exists.

error: 

The property 'succeeded' cannot be found on this object. Verify that the property exists.

At C:\...\...\Documents\WindowsPowerShell\Modules\azure.datafactory.tools\1.8.0\public\Test-AdfLinkedService.ps1:79 char:13
+         if ($null -ne $r -and $r.succeeded) {
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict

**Comment by aam-a** [comment]
Full Error: 

The property 'succeeded' cannot be found on this object. Verify that the property exists.

At C:\...\...\Documents\WindowsPowerShell\Modules\azure.datafactory.tools\1.8.0\public\Test-AdfLinkedService.ps1:79 char:13
+         if ($null -ne $r -and $r.succeeded) {
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], PropertyNotFoundException
    + FullyQualifiedErrorId : PropertyNotFoundStrict

**Comment by NowinskiK** [comment]
Did you check what your StrictMode is?
Try `Set-StrictMode -Off` and rerun the code.

**Comment by aam-a** [comment]
Thank for the feedback - set , Set-StrictMode -Off 
I am getting the same error

**Comment by aam-a** [comment]
Does anyone have any suggestion of this issue?

**Comment by NowinskiK** [comment]
Hi @aam-a , can you provide more information, for example, log from Azure DevOps agent?

**Comment by aam-a** [comment]
@NowinskiK  - Thank you for the response, -there general no logs to pull. Ultimately I run this via Powershell. 
Can I confirm this Test-AdfLinkedService class actually working ?

**Comment by NowinskiK** [comment]
Yes, it's working and has unit tests cover this command + I checked that on my local machine recently.

**Comment by aam-a** [comment]
Thank you for the feedback, following your confirmation it is working. I have built new test environment, in which it is working accordingly, when run for individual ADF Service. I will now test again in my LIVE environment.  

Noted - Same error is generated for all but the first one, if I do multiple ADF Services (separated by comma).

**Comment by NowinskiK** [comment]
What do you mean by "ADF Service" here? Is ADF Service = ADF instance (cloud) or ADF Linked Service? (I hope the latter one).
Can you attach the log? Even if you run via PowerShell & Azure DevOps agent - you can download log content.

**Comment by aam-a** [comment]
To confirm,, as per the class I am using, I meant ADF Linked Service(s). 
Though it is a test env. There is still some sensitive data attached, will double check the full content of the logs, I will then attach accordingly if possible  , thank you for the feedback

**Comment by NowinskiK** [comment]
Of course, remove all sensitive information before sharing. 
Alternatively, you can prepare PowerShell script along with additional files which run all required steps to prove what's wrong/happening, including: deploy ADF with objects, run tests.

**Comment by aam-a** [comment]
As discussed there are no additional logs.
When trying to run the script separating the LinkedServices with comma, i.e. 
 ----
$LinkedServiceName = 'LinkedServiceName',  'LinkedServiceName1', 'LinkedServiceName2'
$r = Test-AdfLinkedService @params -LinkedServiceName $LinkedServiceName
$r | Format-Table
---

It generates the following error: 

Test-AdfLinkdedService: Cannot process argument transformation on paramter 'LinkedServiceName'. Cannot convert value to type System.String
At C:\....... ps1:28 char:55
= ... = Test-AdfLinkdedService @paramas -LinkedServiceName $LinkedServiceName
      + CategoryInfo : InvalidData: (:) [Test-AdfLinkdedService], ParameterBindingArgumentTransformationException
      + FullyQualifiedErrorId : ParameterBindingArgumentTransformationException,Test-AdfLinkdedService

**Please note the script work fine, if do each LinkedService individually ***
